DiamondLightSource/python-artemis
# Adds script to make it easier to test when there is no beam

Fixes #523

See also https://github.com/DiamondLightSource/python-artemis/wiki/Testing-on-the-Beamline

### To test:
1. Replace line 4 of the script with `trap "echo 'caput BL03I-EA-FDBK-01:THRESHOLDPC_XBPM2 $INIT_VALUE' > test_beam_off" EXIT HUP`
3. Run the script
4. Kill it manually, confirm `test_beam_off` contains the expected output
5. Remove `test_beam_off`
6. Re-run the script
7. Log off, confirm `test_beam_off` contains the expected output
